setup/scripts/run_docker_tests.sh

```diff
@@ -13,12 +13,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Delete all running containers
-docker rm -f $(docker ps -a -q)
+echo "Running tests."
 
-docker build -t forseti/base -f scripts/docker/base . || exit 1
-docker build -t forseti/build -f scripts/docker/forseti --no-cache . || exit 1
-docker run -it -d --name build forseti/build /bin/bash || exit 1
+# Check to see if we're on Travis.
+if [ -z ${TRAVIS+x} ]; then
+    # We are on Travis.
```

Comment:
Comments are backwards, right? -z means not on Travis.

Reply:
Really the order of the if should be the same as in other files. Where the on travis is first, and we run coverage, not on travis is second and we only run tests. So remove -z and switch the blocks around.

Reply:
Nice catch, thanks and done.